script.js

```diff
@@ -1,28 +1,31 @@
-const msgWaiting = document.querySelector(".message-waiting");
-const searchForm = document.getElementById("search-form");
-const booksDisplay = document.getElementById("books-display");
+const $msgWaiting = $(".message-waiting");
+const $searchForm = $("#search-form");
+const $booksDisplay = $("#books-display");
 
 const googleBooksURL = "https://www.googleapis.com/books/v1/volumes?q=";
 const apiKey = "&key=AIzaSyCw6aeDwd2srAlaiVBFSSBvyl2O7atCTaA";
 
-searchForm.addEventListener("submit", () => {
-  msgWaiting.style.display = "none";
-  const searchInput = document.getElementById("search-input").value;
+$searchForm.on("submit", () => {
+  $msgWaiting.hide();
+  const searchInput = $("#search-input").val();
```

Comment:
Because we're working with jQuery obejcts, we need to use the jQuery `val` function to grab the value of the input.